### DISCUSSION TITLE
How to build courses in the form of 1 instructor with many students

### DISCUSSION THREAD

**Discussion by Serena1227** [ls1intum/Artemis#7341] [Q&A]
Hi guys, please help me. I need to build courses in the form of:
- Each course has many instructors
- And each instructor has many diff students 
Please guide me how to build, separate / divide, and manage them. Thank you 🙏🏻

**Comment by MaisiKoleni** [comment]
I'm not sure what the problem is, first suggestions:
- if the instructors share the same course materials having one course with all instructors and students should work (classic setup)
- if the instructors don't have a shared approach to teaching the course, each could manage their own course with just their students (unconventional course-per-instructor)

Almost always the first version is the best and most natural, but my intuition says that then you wouldn't ask this question.

Does this help? If not, we will need more details about the course setup and responsibilities.
